### PR TITLE
How to update docs - doc

### PR DESCRIPTION
**Pull request by pwittrock**
## Contributor guidelines
1. Please read our [contributor guidelines](https://github.com/kubernetes/kubernetes/blob/master/CONTRIBUTING.md).
2. See our [developer guide](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/development.md).
3. Follow the instructions for [labeling and writing a release note for this PR](https://github.com/kubernetes/kubernetes/blob/master/docs/devel/pull-requests.md#release-notes) in the block below. (_this would also contain updated detail later about this new template_)

``` release-note
* Use the release-note-* labels to set the release note state 
* Clear this block to use the PR title as the release note 
-OR-
* Enter your extended release note here
```

### CODE REVIEW

**Comment by pwittrock** [comment]
cc @kubernetes/docs

**Comment by pwittrock** [comment]
cc @kubernetes/docs 
cc @janetkuo

File: docs/devel/updating-docs-for-feature-changes.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# How to update docs for new kubernetes features
```
Comment (brendandburns) [inline docs/devel/updating-docs-for-feature-changes.md]: Can you add an overview of what this page is supposed to be about?  By the title it seems like it is how to update the docs, but that's actually just a link here to a different doc.

The other sections are kind of ad-hoc and not really fleshed out in terms of how someone reading _this_ doc should use them,

Comment (pwittrock) [inline docs/devel/updating-docs-for-feature-changes.md]: docs-doc is not well written :(

Thanks Brendan.  Let me know if you think what I have now is sufficient.

**Comment by pwittrock** [comment]
PTAL

**Comment by pwittrock** [comment]
Just updated with munges

**Comment by brendandburns** [comment]
LGTM.

Thanks

**Pull request merged**